.buildkite/ftr_configs.yml

```diff
@@ -401,8 +401,6 @@ enabled:
   - x-pack/test/security_functional/login_selector.config.ts
   - x-pack/test/security_functional/oidc.config.ts
   - x-pack/test/security_functional/saml.config.ts
-  - x-pack/test/security_functional/saml.http2.config.ts
```

Comment:
Can you move these to the disabled section, and add a comment?  There's a lint check that's going to fail.

Reply:
Done ✅